src/Search/SingleHackerModal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+
+import { Flex } from '@rebass/grid';
+import Modal from 'react-modal';
+
+import { IHacker } from '../config';
+import { Button } from '../shared/Elements';
+import SingleHackerView from './SingleHackerView';
+
+interface IModalProps {
+  hacker: IHacker;
+}
+interface IModalState {
+  showModal: boolean;
+}
+
+class SingleHackerModal extends React.Component<IModalProps, IModalState> {
+  constructor(props: IModalProps) {
+    super(props);
+    this.state = {
+      showModal: false,
+    };
+  }
+
+  public render() {
+    return (
+      <div>
+        <Button type="button" onClick={this.handleOpenModal}>
+          See more
+        </Button>
+        <Modal
+          appElement={document.getElementById('root') || undefined}
+          style={{ overlay: { zIndex: 3 } }}
```

Comment:
Can we make this a `styled-component`?

Reply:
https://github.com/reactjs/react-modal/issues/603#issuecomment-378847885

I was having issues before but it seems like there's a workaround for it. I'll try plugging that in and getting back to you.